Gemfile

```diff
@@ -2,3 +2,11 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in redis_cluster.gemspec
 gemspec
+
+group :development do
+  platforms :mri do
+    if RUBY_VERSION >= "2.0.0"
+      gem "pry-byebug"
+    end
+  end
+end
```

Comment:
I ended up putting this in the `Gemfile` instead of the gemspec because Byebug can't install on JRuby or Ruby 1.9.3 (as far as a I know, a gemspec doesn't allow you to dictate these sorts of constraints).

I could potentially clean this up further by moving all the development dependencies to the `Gemfile` if you like.